releasenotes/notes/1.3/pauli-evo-plugins-612850146c3f7d49.yaml

```diff
@@ -15,18 +15,18 @@ features_quantum_info:
 
 features_synthesis:
   - |
-    Added :meth:`.ProductFormula.expand` which allows to view the expansion of a product formula
+    Added :meth:`.ProductFormula.expand`, which lets you view the expansion of a product formula
     in a sparse Pauli format.
```

Comment:
Do you think an example here would be useful?

Reply:
```suggestion
    Added :meth:`.ProductFormula.expand`, which lets you view the expansion of a product formula
    in a sparse Pauli format. For example, we can query the format of a second-order Trotter expansion
    of a Hamiltonian as::
    
        from qiskit.quantum_info import SparsePauliOp
        from qiskit.circuit.library import PauliEvolutionGate
        from qiskit.synthesis.evolution import SuzukiTrotter
        
        hamiltonian = SparsePauliOp(["IX", "XI", "ZZ"], coeffs=[-1, -1, 1])
        evo = PauliEvolutionGate(hamiltonian, time=3.14)
        trotter = SuzukiTrotter(order=2)
        print(trotter.expand(evo))
        
    which will print
    
    .. parsed-literal::
    
        [('X', [0], -3.14), ('X', [1], -3.14), ('ZZ', [0, 1], 6.28), 
        ('X', [1], -3.14), ('X', [0], -3.14)]

```